crt_portal/cts_forms/views.py

```diff
@@ -13,7 +14,17 @@
 @login_required
 def IndexView(request):
     latest_reports = Report.objects.order_by('-create_date')
-    return render_to_response('forms/index.html', {'data_dict': latest_reports})
+    data = []
+    # formatting protected class
+    for l in latest_reports:
```

Comment:
Small thing, but opportunity to use a semantic variable name instead of single letters here.